src/BlockInput.ts

```diff
@@ -342,18 +342,19 @@ export type FieldAny =
   | MathopMenu
   | FrontBackMenu
   | ForwardBackwardMenu
-  | CostumeNumberName;
+  | CostumeNumberName
+  | Variable
+  | List;
 
-/* eslint-disable @typescript-eslint/ban-types */
 export type Any =
+  /* eslint-disable @typescript-eslint/ban-types */
   | Number
   | String
   | Boolean
+  /* eslint-enable @typescript-eslint/ban-types */
   | Angle
   | Color
   | Broadcast
-  | Variable
-  | List
```

Comment:
It was a result of changing `translateFields` in the SB3 I/O to use a `Record<string, BlockInput.FieldAny>`, which made the compiler error because `Variable` and `List` weren't in `FieldAny`.